src/stylesheets/settings/_settings-color.scss

```diff
@@ -18,57 +18,57 @@ Theme palette colors
 */
 
 // Base colors
-$theme-color-base-family:          'gray' !default;
-$theme-color-base-lightest:        $theme-color-base-family, 2 !default;
-$theme-color-base-lighter:         $theme-color-base-family, 5 !default;
-$theme-color-base-light:           $theme-color-base-family, 10 !default;
-$theme-color-base:                 $theme-color-base-family, 30 !default;
-$theme-color-base-dark:            $theme-color-base-family, 50 !default;
-$theme-color-base-darker:          $theme-color-base-family, 70 !default;
-$theme-color-base-darkest:         $theme-color-base-family, 90 !default;
-$theme-color-ink:                  $theme-color-base-darkest !default;
+$theme-color-base-family:           'gray' !default;
+$theme-color-base-lightest:         '#{$theme-color-base-family}-2' !default;
+$theme-color-base-lighter:          '#{$theme-color-base-family}-5' !default;
+$theme-color-base-light:            '#{$theme-color-base-family}-10' !default;
+$theme-color-base:                  '#{$theme-color-base-family}-30' !default;
+$theme-color-base-dark:             '#{$theme-color-base-family}-50' !default;
+$theme-color-base-darker:           '#{$theme-color-base-family}-70' !default;
+$theme-color-base-darkest:          '#{$theme-color-base-family}-90' !default;
+$theme-color-base-ink:              '#{$theme-color-base-family}-90' !default;
 
 // Primary colors
-$theme-color-primary-family:       'blue' !default;
-$theme-color-primary-lightest:     false !default;
-$theme-color-primary-lighter:      $theme-color-primary-family, 10 !default;
-$theme-color-primary-light:        $theme-color-primary-family, 30 !default;
-$theme-color-primary:              $theme-color-primary-family, 60, vivid !default;
-$theme-color-primary-vivid:        'blue-warm', 60, vivid !default;
-$theme-color-primary-dark:         'blue-warm', 70, vivid !default;
-$theme-color-primary-darker:       'blue-warm', 80, vivid !default;
-$theme-color-primary-darkest:      false !default;
+$theme-color-primary-family:        'blue' !default;
+$theme-color-primary-lightest:      false !default;
+$theme-color-primary-lighter:       '#{$theme-color-primary-family}-10' !default;
+$theme-color-primary-light:         '#{$theme-color-primary-family}-30' !default;
+$theme-color-primary:               '#{$theme-color-primary-family}-60v' !default;
+$theme-color-primary-vivid:         'blue-warm-60v' !default;
+$theme-color-primary-dark:          'blue-warm-70v' !default;
+$theme-color-primary-darker:        'blue-warm-80v' !default;
```

Comment:
Why don't these have the `'#{$theme-color-primary-family}-80v'` pattern? Aren't we overriding them in the theme file? Wouldn't that be more abstracted / reuasable to keep these clean and consistent?